src/framework/mpe/internal/stringutils.h

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
I believe needs another name for this file (we already have a file with the name `stringutils.h`), like `articulationstrings.h`

Reply:
makes sense to me, renamed